src/electron/routing_service.ts

```diff
@@ -324,7 +324,7 @@ async function installLinuxRoutingServices(): Promise<void> {
     installationFileDescriptors
       .map(({filename, sha256}) => `/usr/bin/echo "${sha256}  ${path.join(tmp, filename)}" | /usr/bin/shasum -a 256 -c`)
       .join(' && ');
-  command += ` && "${path.join(tmp, LINUX_INSTALLER_FILENAME)}"`;
+  command += ` && "${path.join(tmp, LINUX_INSTALLER_FILENAME)}" "${userInfo().username}"`;
```

Comment:
You don't really need to pass the user name. The script can call `id -un` or I think use `$USER`. Unless you want the script to be able to install for any user.

Reply:
The script/command will be called under root permission, therefore the `$USER` or `id -un` will be `root`.